datajoint/user_relations.py

```diff
@@ -2,59 +2,83 @@
 Hosts the table tiers, user relations should be derived from.
 """
 
-import abc
 from .base_relation import BaseRelation
 from .autopopulate import AutoPopulate
 from .utils import from_camel_case
 from . import DataJointError
 
 
-class Part(BaseRelation, metaclass=abc.ABCMeta):
+class classproperty:
```

Comment:
perhaps it is because there might be a confusion with properties of the metaclass.  I would like to find a discussion explaining this.